arviz/plots/bpvplot.py

```diff
@@ -0,0 +1,294 @@
+        0.94.
+    color : str
+        Matplotlib color
+    figsize : tuple
+        Figure size. If None it will be defined automatically.
+    textsize : float
+        Text size scaling factor for labels, titles and lines. If None it will be
+        autoscaled based on figsize.
+    data_pairs : dict
+        Dictionary containing relations between observed data and posterior/prior predictive data.
+        Dictionary structure:
+
+        - key = data var_name
+        - value = posterior/prior predictive var_name
+
+        For example, `data_pairs = {'y' : 'y_hat'}`
+        If None, it will assume that the observed data and the posterior/prior
+        predictive data have the same variable name.
+    var_names : list of variable names
+        Variables to be plotted, if `None` all variable are plotted. Prefix the variables by `~`
+        when you want to exclude them from the plot.
+    filter_vars : {None, "like", "regex"}, optional, default=None
+        If `None` (default), interpret var_names as the real variables names. If "like",
+        interpret var_names as substrings of the real variables names. If "regex",
+        interpret var_names as regular expressions on the real variables names. A la
+        `pandas.filter`.
+    coords : dict
+        Dictionary mapping dimensions to selected coordinates to be plotted.
+        Dimensions without a mapping specified will include all coordinates for
+        that dimension. Defaults to including all coordinates for all
+        dimensions if None.
+    flatten : list
+        List of dimensions to flatten in observed_data. Only flattens across the coordinates
+        specified in the coords argument. Defaults to flattening all of the dimensions.
+    flatten_pp : list
+        List of dimensions to flatten in posterior_predictive/prior_predictive. Only flattens
+        across the coordinates specified in the coords argument. Defaults to flattening all
+        of the dimensions. Dimensions should match flatten excluding dimensions for data_pairs
+        parameters. If flatten is defined and flatten_pp is None, then `flatten_pp=flatten`.
+    legend : bool
+        Add legend to figure. By default True.
+    ax : numpy array-like of matplotlib axes or bokeh figures, optional
+        A 2D array of locations into which to plot the densities. If not supplied, Arviz will create
+        its own array of plot areas (and return it).
+    backend : str, optional
+        Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
+    plot_ref_kwargs :  dict, optional
+        Extra keyword arguments to control how reference is represented. Passed to `plt.plot` or
+        `plt.axhspan`(when `kind=u_value` and `reference=analytical`).
+    backend_kwargs : bool, optional
+        These are kwargs specific to the backend being used. For additional documentation
+        check the plotting method of the backend.
+    group : {"prior", "posterior"}, optional
```

Comment:
along the idea of comments above, this should be inverted to show the default first